src/lib/is-active.ts

```
@@ -0,0 +1,10 @@
+export function isActive(
+    url: string,
+    pathname: string,
+    nested = true,
+): boolean {
+    if (url.endsWith('/')) url = url.slice(0, -1);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)